package/src/components/ui/Select/styles.ts

```diff
@@ -0,0 +1,247 @@
+  listStyle: 'none',
+  display: 'none',
+  maxHeight: '15em',
+  overflowY: 'auto',
+  border: '0.05em solid #777',
+  borderRadius: '0.25em',
+  width: '100%',
+  left: 0,
+  top: 'calc(100% + 0.35em)',
+  backgroundColor: 'white',
+  zIndex: 100,
+  '&.show': {
+    display: 'block',
+  },
+  variants: {
+    color: {
+      dark: {
+        backgroundColor: _DarkBackgroundColor,
+      },
+      light: {
+        backgroundColor: _LightBackgroundColor,
+      },
+    },
+  },
+  defaultVariants: {
+    color: 'light',
+  },
+});
+
+export const Option = styled('li', {
+  padding: '0.25em 0.5em',
+  textAlign: 'left',
+  cursor: 'pointer',
+  borderRadius: '0.25em',
+  display: 'flex',
+  gap: '0.5em',
+  justifyContent: 'flex-start',
+  alignItems: 'center',
+  variants: {
+    color: {
+      dark: {
+        '&.highlighted': {
+          backgroundColor: 'rgba(255, 255, 255, 0.3)',
+          color: 'white',
+        },
+      },
+      light: {
+        '&.highlighted': {
+          backgroundColor: 'rgba(0, 0, 255, 0.7)',
+          color: 'white',
+        },
+      },
+    },
+  },
+  defaultVariants: {
+    color: 'light',
+  },
+});
+
+export const OptionBadge = styled('button', {
+  display: 'flex',
+  alignItems: 'center',
+  border: '0.05em solid #777',
+  borderRadius: '0.25em',
+  padding: '0.15em 0.25em',
+  gap: '0.25em',
+  cursor: 'pointer',
+  background: 'none',
+  outline: 'none',
+  '&:hover': {
+    '#RB': {
+      color: 'hsl(0, 100%, 50%)',
+    },
+  },
+  '&:focus': {
+    '#RB': {
+      color: 'hsl(0, 100%, 50%)',
+    },
+  },
+  variants: {
+    color: {
+      dark: {
+        color: 'white',
+        '&:hover': {
+          backgroundColor: 'rgba(255, 255, 255, 0.3)',
+          borderColor: 'hsl(0, 100%, 50%)',
+        },
+        '&:focus': {
+          backgroundColor: 'rgba(255, 255, 255, 0.3)',
+          borderColor: 'hsl(0, 100%, 50%)',
+        },
+      },
+      light: {
+        color: 'black',
+        '&:hover': {
+          backgroundColor: 'hsl(0, 100%, 90%)',
+          borderColor: 'hsl(0, 100%, 50%)',
+        },
+        '&:focus': {
+          backgroundColor: 'hsl(0, 100%, 90%)',
+          borderColor: 'hsl(0, 100%, 50%)',
+        },
+      },
+    },
+  },
+});
+
+export const RemoveButton = styled('span', {
+  fontSize: '1.2em',
+  color: '#777',
+  marginTop: '0.1em',
+});
+
+export const PulseContainer = styled('div', {
+  height: '0.5em',
+  width: '0.5em',
+  marginLeft: 'auto',
+  position: 'relative',
+});
+
+export const StaticPulse = styled('div', {
+  height: '100%',
+  width: '100%',
+  borderRadius: '999px',
+  zIndex: '100',
+  variants: {
+    status: {
+      pending: {
+        backgroundColor: 'yellow',
+      },
+      confirm: {
+        backgroundColor: 'green',
+      },
+    },
+  },
+});
+
+const scaleUp = keyframes({
+  '0%, 50%': {
+    transform: 'scale(1)',
+    opacity: '0',
+  },
+  '100%': {
+    transform: 'scale(1.5)',
+    opacity: '1',
+  },
+});
+
```

Comment:
Can we remove the scaling effect for the current chain which is _green_ in color? Imo would be great if we have a scaling impact only for the _yellow_ color pending indicator